Client/Coordinators/SettingsCoordinator.swift

```diff
@@ -155,6 +155,34 @@ class SettingsCoordinator: BaseCoordinator, SettingsDelegate, SettingsFlowDelega
         router.push(creditCardViewController)
     }
 
+    // TODO: Move the both show password methods into it's own coordinator
+    func showPasswordList() {
+        let navigationHandler: (_ url: URL?) -> Void = { [weak self] url in
+            guard let url = url else { return }
+            self?.settingsOpenURLInNewTab(url)
```

Comment:
I'm curious if we can move the guard upward in the call chain and have the method accept an honest non optional URL since the else case just abandons everything in the method.

Reply:
Ideally yes but the interface needs to remain the same while the feature flag is in use. Also I think once the entire flow is converted to coordinators this closure won't be needed anymore because the VC can just call the coordinator delegate directly.